.github/workflows/e2e-quantic.yml

```diff
@@ -44,12 +44,25 @@ jobs:
 
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
       - uses: ./.github/actions/setup
-      - uses: ./.github/actions/e2e-quantic
+      - uses: ./.github/actions/e2e-quantic-cypress
         with:
           spec: ${{ matrix.spec }}
+  e2e-quantic-playwright-test:
+    name: 'Run Playwright e2e tests on Quantic'
+    needs: e2e-quantic-cypress-test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+        with:
+          egress-policy: audit
+
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+      - uses: ./.github/actions/setup
+      - uses: ./.github/actions/e2e-quantic-playwright
   e2e-quantic-cleanup:
     if: cancelled() || failure() || success()
-    needs: e2e-quantic-test
+    needs: e2e-quantic-playwright-test
```

Comment:
doesnt it also needs the e2e-quantic-cypress? or because playwright relies on the e2e-quantic-cypress-test, its ok ? seems a bit confusing that playwright needs cypress no?

Reply:
Yes, this works by transitivity!

and yeah the playwrights needs the cypress test cause we want them to run after cause I'm afraid that it will cause a mess if we run all our Cypress tests and all our Playwright tests in the same org in the same time,

and this is temporary, cause as soon as we finish our playwright tests we will delete the cypress part.